kadasrouting/gui/navigationpanel.py

```diff
@@ -201,6 +207,8 @@ def updateNavigationInfo(self, gpsinfo):
         iface.mapCanvas().refresh()
         self.rubberband.reset(QgsWkbTypes.LineGeometry)
 
+        self.gpsConnection.statusChanged.connect(self.updateNavigationInfo)
```

Comment:
Is this needed? This connection is already made here:

https://github.com/camptocamp/kadas-routing-plugin/pull/9/files#diff-569c8efd67323cd7956994011be4a0faL364

Reply:
yes, because I disconnect it on line 193 above to avoid an infinite loop